src/attesters/csv/collect_evidence.c

```diff
@@ -144,6 +157,26 @@ static int csv_get_hsk_cek_cert(const char *chip_id, csv_evidence *evidence_buff
 	return 0;
 }
 
+/**
+ * Download HSK and CEK cert, and then save them to @hsk_cek_cert.
+ *
+ * Params:
+ * 	hsk_cek_cert [in]: the buffer to save HSK and CEK cert
```

Comment:
Thank you very much! The param is evidence_buffer and chip_id. Could you please fix it?